test/e2e/specs/customer_admin.go

```diff
@@ -1,34 +1,53 @@
-//+build e2e
-
-package e2e
+package specs
 
 import (
 	"errors"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
 	rbacv1 "k8s.io/api/rbac/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
 )
 
-var _ = Describe("Openshift on Azure customer-cluster-admin e2e tests [CustomerAdmin]", func() {
-	defer GinkgoRecover()
+var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin]", func() {
+	var (
+		cli      *openshift.Client
+		admincli *openshift.Client
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = openshift.NewSystemAdminClient()
+		if err != nil {
+			Skip(err.Error())
+		}
+		admincli, err = openshift.NewCustomerAdminClient()
+		if err != nil {
+			Skip(err.Error())
+		}
+	})
 
 	It("should not read nodes", func() {
-		_, err := cadmin.kc.CoreV1().Nodes().Get("master-000000", metav1.GetOptions{})
+		_, err := admincli.CoreV1.Nodes().Get("master-000000", metav1.GetOptions{})
 		Expect(kerrors.IsForbidden(err)).To(Equal(true))
 	})
 
 	It("should have full access on all non-infrastructure namespaces", func() {
 		// Create project as normal user
-		namespace := nameGen.generate("e2e-test-")
-		c.createProject(namespace)
+		namespace, err := randomstring.RandomString("abcdefghijklmnopqrstuvwxyz0123456789", 5)
+		Expect(err).ToNot(HaveOccurred())
+		namespace = "e2e-test-" + namespace
+		cli.CreateProject(namespace)
```

Comment:
The point of using projects in e2e tests was to avoid using the admin kubeconfig. Why is it needed now?